python/pyarrow/_s3.pyx

```diff
@@ -0,0 +1,100 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Nit, but I'd rather call this file `_s3fs.pyx`.

Reply:
Updated